ENGAGEHF.xcodeproj/project.pbxproj

```diff
@@ -1313,6 +1587,14 @@
 				minimumVersion = "3.0.0-beta.1";
 			};
 		};
+		4DC0F6722C34929C0025AB13 /* XCRemoteSwiftPackageReference "ExceptionCatcher" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/sindresorhus/ExceptionCatcher.git";
```

Comment:
We should use a tagged version up to the next major version, e.g., https://github.com/sindresorhus/ExceptionCatcher/releases/tag/v2.0.1

Reply:
I actually ended up moving away from initializing HKUnits from a string, which is why this package was included. We can safely remove the dependency entirely.